src/dtgtk/thumbtable.c

```diff
@@ -2017,6 +2042,54 @@ static gboolean _accel_color(GtkAccelGroup *accel_group, GObject *acceleratable,
 {
   GList *imgs = g_list_copy((GList *)dt_view_get_images_to_act_on(FALSE, TRUE));
   dt_colorlabels_toggle_label_on_list(imgs, GPOINTER_TO_INT(data), TRUE);
+
+  // if we are in darkroom we show a message as there might be no other indication
+  const dt_view_t *v = dt_view_manager_get_current_view(darktable.view_manager);
+  if(v->view(v) == DT_VIEW_DARKROOM && g_list_length(imgs) == 1 && darktable.develop->preview_pipe)
+  {
+    // we verify that the image is the active one
+    const int id = GPOINTER_TO_INT(g_list_nth_data(imgs, 0));
+    if(id == darktable.develop->preview_pipe->output_imgid)
+    {
+      GList *res = dt_metadata_get(id, "Xmp.darktable.colorlabels", NULL);
+      res = g_list_first(res);
+      gchar *result = NULL;
+      if(res != NULL)
+      {
+        do
+        {
+          const char *lb = (char *)(dt_colorlabels_to_string(GPOINTER_TO_INT(res->data)));
+          if(g_strcmp0(lb, "red") == 0)
+          {
+            result = dt_util_dstrcat(result, "<span foreground=\"#ee0000\">⚫ </span>");
+          }
+          else if(g_strcmp0(lb, "yellow") == 0)
+          {
+            result = dt_util_dstrcat(result, "<span foreground=\"#eeee00\">⚫ </span>");
+          }
+          else if(g_strcmp0(lb, "green") == 0)
+          {
+            result = dt_util_dstrcat(result, "<span foreground=\"#00ee00\">⚫ </span>");
+          }
+          else if(g_strcmp0(lb, "blue") == 0)
+          {
+            result = dt_util_dstrcat(result, "<span foreground=\"#0000ee\">⚫ </span>");
+          }
+          else if(g_strcmp0(lb, "purple") == 0)
+          {
+            result = dt_util_dstrcat(result, "<span foreground=\"#ee00ee\">⚫ </span>");
+          }
```

Comment:
This is great, but there's a problem we've dealt with back when similar code was used for info line. Basically if user has colormoji font (or something like that), they'll see just  "colorlabels set to :black_circle: :black_circle: :black_circle: ".

Maybe add color name in parenthesis for those?
eg: "colorlabels set to :black_circle: (red) :black_circle: (green) :black_circle: (blue)".

for people who don't have colormoji font that would look like:
"colorlabels set to :red_circle: (red) :green_circle: (green) :large_blue_circle: (blue)".

Reply:
Or use a standard symbol char like : ●

Reply:
Or a larger version ⬤

The point is that using standard symbols will work in all cases.